OrbitService/ProcessList.cpp

```diff
@@ -43,55 +41,20 @@ ErrorMessageOr<void> ProcessList::Refresh() {
 
     auto iter = processes_map_.find(pid);
     if (iter != processes_map_.end()) {
-      ProcessInfo& process(*(iter->second));
+      Process& process(*(iter->second));
```

Comment:
it looks a bit weird - this initialized reference not an object correct? maybe use = here?

Reply:
I agree. done.